### PR TITLE
Rule S4653: Units should be valid

### LINKED ISSUE #19: Rule S4653: Units should be valid
Implements [RSPEC-4653](https://jira.sonarsource.com/browse/RSPEC-4653)

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #19

### CODE REVIEW

File: its/plugin/projects/issues-project/src/file1.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
   background-color: #ffw; /* issue here */
   color: #2d5e8b;
   foo: "some text";
+  width: 100pixels;
```
Comment (vilchik-elena) [inline its/plugin/projects/issues-project/src/file1.css]: let's have comment with rule key (both stylelint and SQ)

File: its/plugin/projects/issues-project/src/file1.css [hunk 2]
```diff
@@ -5,6 +5,7 @@
...
 }
 
 /* some comment */
```
Comment (vilchik-elena) [inline its/plugin/projects/issues-project/src/file1.css]: we can probably remove comments and everything not required to raise the issues

File: sonar-css-plugin/src/main/java/org/sonar/css/plugin/CssRules.java [hunk 3]
```diff
@@ -46,12 +43,6 @@ public CssRules(CheckFactory checkFactory) {
     }
   }
 
-  public static List<Class> getRuleClasses() {
-    return Collections.unmodifiableList(Arrays.asList(
-      ColorNoInvalidHex.class
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/main/java/org/sonar/css/plugin/CssRules.java]: IMO it makes more sense to keep the list of rule classes in this CssRules class

File: sonar-css-plugin/src/test/java/org/sonar/css/plugin/SonarWayProfileTest.java [hunk 4]
```diff
@@ -39,6 +39,7 @@ public void should_create_sonar_way_profile() {
     assertThat(profile.name()).isEqualTo(SonarWayProfile.PROFILE_NAME);
     assertThat(profile.rules()).extracting("repoKey").containsOnly(CssRulesDefinition.REPOSITORY_KEY);
     assertThat(profile.rules()).extracting("ruleKey").contains("S4647");
+    assertThat(profile.rules()).extracting("ruleKey").contains("S4653");
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/test/java/org/sonar/css/plugin/SonarWayProfileTest.java]: you don't need to assert it for every rule, probably size assertion should be enough (as all rules should be activated)

File: sonar-css-plugin/src/test/java/org/sonar/css/plugin/TokenizerTest.java [hunk 5]
```diff
@@ -22,6 +22,8 @@
 import java.util.List;
 import org.junit.Test;
 
+import java.util.List;
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/test/java/org/sonar/css/plugin/TokenizerTest.java]: @Evangelink why this commit is here?

Comment (Evangelink) [reply sonar-css-plugin/src/test/java/org/sonar/css/plugin/TokenizerTest.java]: No clue... I guess the automatic merge when rebasing did not worked properly.

**Pull request merged**